examples/triton-docker-combined/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "triton_account" {}
+variable "triton_url" {
+  default = "https://us-sw-1.api.joyentcloud.com"
+}
+variable "triton_key_path" {
+  default = "path to your ssh key for triton"
+}
+variable "triton_key_id" {}
```

Comment:
I like that you're putting instructions in the defaults elsewhere. Perhaps this one could use the same treatment?